dts/arm/nordic/nrf52840.dtsi

```diff
@@ -32,6 +32,15 @@
 			};
 	};
 
+	emulated-flash-controller@0 {
```

Comment:
insert this into #if defined(CONFIG_FLASH_EMULATOR) ...#endif.

I expect similar addition to the qemu boards dts files.